<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>808</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Titillium Web</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Число папок</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Число файлов</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Folder_count_line">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="File_count_line">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущий путь</string>
    </property>
   </widget>
   <widget class="QLabel" name="Current_path_label">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Нет пути</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Change_path_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сменить папку</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>250</y>
      <width>601</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1334</width>
     <height>49</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
